CONTRIBUTING.md

```diff
@@ -1,145 +1,118 @@
+## Code of Conduct
 
-Each commit message consists of a **header**, **body** and **footer**.
+This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT.md). By contributing, you are expected to honor these guidelines.
 
-#### Header
+## Developing
 
-The header is required and must not exceed 70 characters to ensure it is well-formatted in common git tools. It has a special format that includes a *type*, *scope* and *subject*:
+### Installation
 
-Syntax:
+1. Fork the repository `one-app-dev-proxy` to your GitHub account.
+2. Afterwards run the following commands in your terminal
 
-```bash
-<type>(<scope>): <subject>
-```
+    ```bash
+    $ git clone https://github.com/<your-github-username>/one-app-dev-proxy
+    $ cd one-app-dev-proxy
+    ```
 
-#### Type
+   > replace `your-github-username` with your github username
 
-The *type* should always be lowercase as shown below.
+3. Install the dependencies by running
 
-##### Allowed `<type>` values:
+    ```bash
+    $ cd npm install
+    ```
 
-* **feat** (new feature for the user)
-* **fix** (bug fix for the user, not a fix to build scripts)
-* **docs** (changes to documentation)
-* **style** (formatting, missing semi colons, etc; no functional code change)
-* **refactor** (refactoring production code, eg. renaming a variable)
-* **test** (adding missing tests, refactoring tests; no production code change)
-* **chore** (updating build/env/packages, etc; no production code change)
+4. You can now run any of these scripts from the root folder.
 
-#### Scope
+#### Generating the build files
 
-The *scope* describes the affected code. The descriptor may be a route, component, feature, utility, etc. It should be one word or camelCased, if needed:
+`npm run build`
 
-```bash
-feat(transactions): added column for quantity
-feat(BalanceModule): initial setup
-```
+Builds the module into the `lib` folder.
 
-The commit headers above work well if the commit affects many parts of a larger feature. If changes are more specific, it may be too broad. To better clarify specific scopes, you should use a `feature/scope` syntax:
+#### Running tests
 
-```bash
-fix(transaction/details): missing quantity field
-```
+`npm test`
 
-The above syntax helps reduce verbosity in the _subject_. In comparison, consider the following example:
+Runs `eslint` **and** unit tests on the current branch.
 
-```bash
-fix(transaction): missing quantity field in txn details
-```
+`npm posttest`
 
-Another scenario for scope is using a `route/scope` (or `context/scope`) syntax. This would be useful when a commit only affects a particular instance of code that is used in multiple places.
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
```

Comment:
this just lints the lockfile
https://github.com/americanexpress/one-app-dev-proxy/blob/master/package.json#L13

conventional commits are run as part of git hooks